cpu/die/runtest.sh

```diff
@@ -17,35 +17,32 @@
 # Boston, MA 02110-1301, USA.
 #
 
-# Include the BeakerLib environment
-. /usr/share/beakerlib/beakerlib.sh
-
 FILE=$(readlink -f ${BASH_SOURCE})
 NAME=$(basename $FILE)
 CDIR=$(dirname $FILE)
 TEST=${TEST:-"$0"}
 TMPDIR=/var/tmp/$(date +"%Y%m%d%H%M%S")
 
-source ${CDIR%/cpu/die}/cpu/common/libbkrm.sh
-source ${CDIR%/cpu/die}/cpu/common/libutil.sh
-source ${CDIR%}/../../cki_lib/lib.sh
+#source /usr/bin/rhts_environment.sh
+#source /usr/share/beakerlib/beakerlib.sh
```

Comment:
Could we remove these two commented-out lines?

Reply:
Removed.